styles/pages/statistics.module.scss

```diff
@@ -0,0 +1,18 @@
+.main {
+	flex-grow: 1;
+	display: flex;
+	align-content: flex-start;
+	flex-wrap: wrap;
+	padding-block: 3.5vw;
```

Comment:
Making the flex box wrapping its elements is meaningless here. If you shrink the stats as the viewports shrinks, they will never wrap.

The intended behavior here is that if the width of the viewport shrinks, the only thing that will happen is the wrap (if there's an overflow) - not the shrinking of the stats

This, therefore, applies to the stat module aswell.

Reply:
noted

Reply:
- [ ] The texts in the stat should be propertional to the parent' dimensions. 
- [ ] The stats must always be squares (same height and width)
- [ ] Currently, if the list length is not even, the last stat takes the place of the whole row. This is not the intention it should remain the same size as if it were 1 element out of the 2 in a row.

`display: grid` is good for setting the 2 columns, if you want.

Reply:
noted